lib/rails_admin_loadevent.rb

```diff
@@ -0,0 +1,44 @@
+require 'rails_admin/config/actions'
+require 'rails_admin/config/actions/base'
+ 
+module RailsAdminLoadevent
+end
+ 
+module RailsAdmin
+  module Config
+    module Actions
+      class Loadevent < RailsAdmin::Config::Actions::Base
+	RailsAdmin::Config::Actions.register(self) 
+        register_instance_option :collection do
+          true
+        end
+        register_instance_option :link_icon do
+          'icon-fire'
+        end 
+        #disable jquery pjax
+        register_instance_option :pjax? do
+          false
+        end
+        register_instance_option :controller do
+          Proc.new do
+	    # find all the selected and archived records
+            allSelectedEvents = Proposal.where(selected: "t", archived: "f" ).select("title", "id")
+	    # if records nil do no contnue
+            unless allSelectedEvents.nil?
+             allSelectedEvents.each { |eachEvent|
+	       #if records already exist
+              unless Event.exists?(eachEvent.attributes)
+		recordEvent = Event.create(eachEvent.attributes)
+                if recordEvent.valid?
+		  recordEvent.save
+		end
+               end
```

Comment:
I definitely like the idea of switching to all custom code, vs. using rails_admin -- that makes this a much better training project